include/overworld.h

```diff
@@ -119,4 +126,7 @@ bool32 sub_8055C9C(void);
 void Overworld_ResetStateAfterDigEscRope(void);
 bool32 sub_8058244(void);
 
+void sub_80574A4(void);
+bool32 sub_805750C(u8 *, const struct CreditsOverworldCmd *, u8);
+
 #endif //GUARD_ROM4_H
```

Comment:
It seems that someone didn't change the header macro after changing the header file name

Reply:
Fixed.